orion/sites/admin/src/achieve/achieve.js

```diff
@@ -2,13 +2,54 @@
 require('./achieve.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
+import API from '../api.js';
+import { Modal } from '../modals/modal.js';
+import { Link } from 'react-router-dom';
 
 export class AchievePage extends React.Component {
+	constructor(props) {
+    super(props);
```

Comment:
hm looks like your indenting is a bit weird. If you're using Atom, go to top right Atom > Preferences > Editor > scroll down to find Tab Length (Number of spaces to represent a tab) and use 4.

After that make sure your indenting on Atom is consistent, and it should reflect accordingly in Github.